components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -70,7 +70,8 @@ class AddonManager(
                 .map { addon -> addon.filterTranslations(locales) }
                 .map { addon ->
                     installedExtensions[addon.id]?.let {
-                        addon.copy(installedState = it.toInstalledState())
+                        addon.copy(installedState = it.toInstalledState(),
```

Comment:
nit: assign a val to `it.toInstalledState()`

Reply:
👍